hack/push-images.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/python
+
+# Copyright 2014 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# push images to gcr.io
```

Comment:
Is this script useful to keep in the tree? It seems useful as a one-off and possibly if it took arguments, but shouldn't instead we be shifting the build process for at least the k8s/ images to push straight, which means this script might not even be a good idea to run a second time?